apps/judicial-system/backend/src/app/formatters/formatters.ts

```diff
@@ -675,6 +680,7 @@ export const formatDefenderAssignedEmailNotification = (
 
   const body = formatMessage(notifications.defenderAssignedEmail.body, {
     defenderHasAccessToRVG: Boolean(overviewUrl),
+    advocateType,
```

Comment:
I think there be a hole here in regards to the advocates access to RVG. 
The overviewUrl seems to be formatted and sent so long as an advocates national id is provided (in the internalNotificationService), but isn't the legal rights protectors access to the case files dependent on a choice made by the judge and stored in the civil claimant table under case_files_shared_with_spokesperson?

Reply:
Advocates have access to RVG if they are in the lawyer registry. Wether or not they have access to the case files is another thing. In this case, all advocates should have access to RVG because they are listed in the lawyer registry. There they can see a overview of the case but not the case files if the judge selected it as such. 

That logic is currently being worked on in [Bótakrafa - aðgangur réttargæslumanns/lögmanns kröfuhafa að réttarvörslugáttinni](https://app.asana.com/0/1199153462262248/1208167750327275/f).